### ISSUE TITLE
kill value cannot convert int64 to string

### ISSUE THREAD

**Issue by pgold30** [asobti/kube-monkey#139]
When trying to apply examples I got the error
```
cannot convert int64 to string
```

I was able to isolate the issue and the problem is in line 
```  
kube-monkey/kill-value: 1
```

I was also able to fix it like this :
```
kube-monkey/kill-value: '3'
```

But you should change that in the Readme

PR created https://github.com/asobti/kube-monkey/pull/140

**Comment by asobti** [comment]
Interesting catch. I see your PR, but I want to take a look at the code to see if we can fix this by allowing the code to handle both integers and strings.

**Comment by asobti** [comment]
Nevermind. As per the k8s docs, label values must be strings
https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

**Comment by asobti** [comment]
Fixed by #140
